src/DeviceProvisioningServices/DeviceProvisioningServices/Cmdlet/IotDpsDataPlane/Registration/GetAzIotDeviceProvisioningServiceRegistration.cs

```diff
@@ -0,0 +1,110 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Management.DeviceProvisioningServices
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.Management.DeviceProvisioningServices.Models;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Devices.Provisioning.Service;
+    using Microsoft.Azure.Management.DeviceProvisioningServices;
+    using Microsoft.Azure.Management.DeviceProvisioningServices.Models;
+    using Microsoft.WindowsAzure.Commands.Utilities.Common;
+
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "IoTDeviceProvisioningServiceRegistration", DefaultParameterSetName = ResourceParameterSet)]
+    [Alias("Get-" + ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "IoTDpsRegistration")]
+    [OutputType(typeof(PSDeviceRegistrationState), typeof(PSDeviceRegistrationStates[]))]
```

Comment:
It looks like it's not necessary to have two output types

Reply:
This is only to display the output in different format